modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc

```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing. 
+Therefore in a production environment it is recommended to do the same.
+The ownCloud support team requires the reproduction of any potential cases
```

Comment:
The ownCloud support team requires the reproduction of any potential cases
 with mod_php.

Any issues with the ownCloud server have to be reproducible with `mod_php`.